src/GeoDB.cpp

```diff
@@ -127,7 +129,7 @@ std::string MaxmindDB::_getGeoLocString(MMDB_lookup_result_s *lookup) const
     return geoString;
 }
 
-std::string MaxmindDB::getASNString(const struct sockaddr *sa) const
+std::string MaxmindDB::getASNString(const struct sockaddr *sa, bool &error) const
```

Comment:
similar to previous, instead of calling it error should we call it `notfound`? it's not an actual error but rather a status/condition